src/astx/transpilers/python.py

```diff
@@ -62,6 +69,26 @@ def visit(self, node: astx.Block) -> str:
         """Handle Block nodes."""
         return self._generate_block(node)
 
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: astx.ImportFromStmt) -> str:
+        """Handle ImportFromStmt nodes."""
+        names = [self.visit(name) for name in node.names]
+        level_dots = "." * node.level
+        module_str = (
+            f"{level_dots}{node.module}" if node.module else level_dots
+        )
+        names_str = ", ".join(str(name) for name in names)
+        return f"from {module_str} import {names_str}"
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: astx.ImportStmt) -> str:
+        """Handle ImportStmt nodes."""
+        if not isinstance(node.names, list):
```

Comment:
please remove this `isinstance` validation, because it will be done via typeguard